packages/azure-container-services/images/test-images/build.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+#######################################
+# (c) Copyright IBM Corp. 2023
+#######################################
+
+
+# This script builds a test image that can be used as an Azure container service.
+# You can use the Instana Node.js Azure base image from various sources, such as a published production image,
+# an image from your local Docker registry, or an image from an Azure container registry with pre-release images.
+# You would usually not call this script directly but either use ./build-and-push.sh to push the built image
+# to a registry for use in an actual Azure service or use ./build-and-run.sh to run it locally in a simulated Azure environment.
+
+# Parameters:
+# - $1: Instana Layer Mode, which Docker base image layer to use. Options:
+#     - released      -> use an official production image from the public IBM container registry (icr.io)
```

Comment:
Suggestion: We cannot test some of the build steps locally because the images are not yet released. Do we want to test these after the release? We could create a list of things to do after the release.